sfs/time/drivingfunction.py

```diff
@@ -251,7 +252,191 @@ def apply_delays(signal, delays):
     delays_samples = np.rint(samplerate * delays).astype(int)
     offset_samples = delays_samples.min()
     delays_samples -= offset_samples
-    out = np.zeros((delays_samples.max() + len(data), len(delays_samples)))
-    for column, row in enumerate(delays_samples):
-        out[row:row + len(data), column] = data
-    return util.DelayedSignal(out, samplerate, offset_samples / samplerate)
+    out = np.zeros((delays_samples.max() + len(signal), len(delays_samples)))
+    for channel, cdelay in enumerate(delays_samples):
+        out[cdelay:cdelay + len(signal), channel] = signal
+    return out, offset_samples / fs
+
```

Comment:
This will break the current implementation of the time based WFS driving functions.
I think we wanted to try out using the `signal` class, here is an example :  https://github.com/sfstoolbox/sfs-python/blob/master/sfs/util.py#L141

Reply:
Thank you. Probably I chose the wrong script while merging.